data/sql/updates/pending_db_world/rev_1632568882031573700.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632568882031573700');
+
+UPDATE `smart_scripts` SET `action_param3`=990000 WHERE  `entryorguid`=179664 AND `source_type`=1 AND `id`=0 AND `link`=0;
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=14462;
```

Comment:
miss source_type here, you need to delete per entryorguid and per source_type, so add AND source_type = 0

Reply:
@Ragelezz

Reply:
after irl work